### DISCUSSION TITLE
Feature request: Support single site

### DISCUSSION THREAD

**Discussion by rrjanbiah** [lloc/Multisite-Language-Switcher#195] [Q&A] [answered]
Document suggests to setup multisite https://msls.co/user-docs/install-multisite.html Is that mandatory? Can't multilanguage be supported in a single site?

**Comment by lloc** [comment, marked as answer]
I get your point and there are some solutions out there that already fulfil your request. Take [Polylang](https://github.com/polylang/polylang) for example.

The Multisite Language Switcher on the other hand uses WordPress' native feature (the multisite) to have another (maybe more) flexible approach to expose not only 1:1 translations but also cultural differences. For example, it can make sense to have completely different content parts of your websites and in some other features in your website you need translations.

**Reply by rrjanbiah** [reply]
Thank you
